packages/terra-theme-properties/src/scss/semantic-names.scss

```diff
@@ -323,13 +323,14 @@
 
 // =======================================
 // terra-modal-manager
-// https://github.com/cerner/terra-core/tree/main/packages/terra-modal-manager
+// https://github.com/cerner/terra-framework/tree/main/packages/terra-modal-manager
```

Comment:
These files are generated.  I communicated this issue with the UX team.  You don't want to manually update it as it would just be incorrect the next release.

Reply:
Awesome, thank you so much for taking care of that, reverted in https://github.com/cerner/terra-framework/pull/1994/commits/dcf6be070dc631b1296ea771e9b984a402cd7616